.github/workflows/tests.yml

```diff
@@ -72,7 +72,7 @@ jobs:
         shell: bash
         run: .github/mem-constrained-exec.sh
       - name: Run Sonar
-        if: ${{ matrix.os == 'ubuntu-latest' && (github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == 'SpoonLabs/sorald') }}
```

Comment:
That's not a good idea. You can't use secrets in pull requests from forks, so by removing this all PRs from forks will fail.

Reply:
The commit before you removed this safety check did run the Sonar scan, as it matches on the head repo name.

Reply:
I just did this for testing purposes. I am going to revert this eventually.

When I logged into the SonarCloud portal, I couldn't see a report for the scan made for this PR. That is why I tried to force it.

The issue has to do something with the refactoring. Until now, we only needed to register `SpoonLabs_sorald`, but now we have three modules - `sorald_parent`, `sorald`, and `sorald_api` - thanks to the #808 . The scans which I saw throw warnings that they are not able to find the sorald-api JAR.